### PR TITLE
Update set-up-a-custom-do-not-rewrite-urls-list-with-atp.md

### PR DESCRIPTION
**Pull request by msbemba**
Added another step to Turn ON URLs will be rewritten and checked against a list of known malicious links when user clicks on the link

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2424

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/security/office-365-security/set-up-a-custom-do-not-rewrite-urls-list-with-atp.md [hunk 1]
```diff
@@ -55,11 +55,13 @@ To edit (or define) ATP policies, you must be assigned an appropriate role. The
 
 4. Specify a name and description for your policy.
 
-5. In the **Do not rewrite the following URLs** section, select the **Enter a valid URL** box, and then type a URL, and then choose the plus sign (+).
+5. Turn **ON** URLs will be rewritten and checked against a list of known malicious links when user clicks on the link.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/set-up-a-custom-do-not-rewrite-urls-list-with-atp.md:58]:
```suggestion
5. Turn **ON** URLs to allow them to be rewritten and checked against a list of known malicious links when the user clicks on the link.
```

Comment (JohanFreelancer9) [reply microsoft-365/security/office-365-security/set-up-a-custom-do-not-rewrite-urls-list-with-atp.md:58]: I am really not sure about this. Please confirm.

File: microsoft-365/security/office-365-security/set-up-a-custom-do-not-rewrite-urls-list-with-atp.md [hunk 2]
```diff
@@ -55,11 +55,13 @@ To edit (or define) ATP policies, you must be assigned an appropriate role. The
...
 
-6. In the **Applied To** section, choose **The recipient is a member of**, and then choose the group(s) you want to include in your policy. Choose **Add**, and then choose **OK**.
+6. In the **Do not rewrite the following URLs** section, select the **Enter a valid URL** box, and then type a URL, and then choose the plus sign (+).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/set-up-a-custom-do-not-rewrite-urls-list-with-atp.md]:
```suggestion
6. In the **Do not rewrite the following URLs** section, select the **Enter a valid URL** box, enter a URL, and then choose the plus sign (+).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/security/office-365-security/set-up-a-custom-do-not-rewrite-urls-list-with-atp.md (hunk 1 above)
Comment (msbemba) [reply microsoft-365/security/office-365-security/set-up-a-custom-do-not-rewrite-urls-list-with-atp.md:58]: This is a setting in o365 and it states "URLs will be rewritten and checked against a list of known malicious links when user clicks on the link." We are turning it ON. So please confirm if this change is required.

Comment (JohanFreelancer9) [reply microsoft-365/security/office-365-security/set-up-a-custom-do-not-rewrite-urls-list-with-atp.md:58]: Thank you for the explanation. Now it makes sense. Please ignore my suggestion.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrisda Please merge this PR. Thanks.

**Pull request merged**